invenio_rdm_records/fixtures/data/vocabularies.yaml

```diff
@@ -11,3 +11,6 @@ resource_types:
   pid-type: rsrct
   data-file: vocabularies/resource_types.yaml
 # 'subjects' is highly dependent on the kind of InvenioRDM instance, so no defaults are provided.
```

Comment:
Let's reword that comment to explain why an empty subjects file is given rather than no subjects entry at all.

Reply:
I tried to improve it but I am open to suggestions.